src/generate-services.ts

```diff
@@ -20,6 +20,7 @@ import {
 import SourceInfo, { Fields } from './sourceInfo';
 import { contextTypeVar } from './main';
 import { Context } from './context';
+import { grpcWebResponsePromiseOrObservable } from './generate-grpc-web';
```

Comment:
How does this compare to the solution in #629

Reply:
This pull request more than to support grpc-web Promise stream. 

After this pull request, fix incorrect return type( #629 ) can do more simple.

Reply:
I don't understand what you mean by "can do more simple"

Reply:
My question is - why add a new function `grpcWebResponsePromiseOrObservable` instead of adding a parameter to the previous function?

Reply:
I want to support `Promise Stream`, need to add some specifical response type, but this only using in `grpc-web`.

Reply:
Ok, I see.

Please `git fetch` and then `git rebase 557a5a186fbc375f09c3b3ceea14fe502c5ce34d`

This will bring in the regression test to ensure that the issue from #629  is also fixed.

Reply:
Ok, I updated it